python/ray/ml/utils/tensorflow_utils.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import tensorflow as tf
+
+
+def convert_pandas_to_tf_tensor(df: pd.DataFrame) -> tf.Tensor:
+    """Convert a pandas dataframe to a TensorFlow tensor.
```

Comment:
Can you spec out what's expectation on this dataframe (given not all types are convertible)?

Reply:
Fixed